src/main/java/com/parasoft/findings/sonar/ParasoftConfiguration.java

```diff
@@ -40,6 +40,16 @@ public static List<PropertyDefinition> getProperties()
                 .multiValues(true)
                 .build()
         );
+        // Support SOAtest Test Execution
+        items.add(PropertyDefinition.builder(ParasoftConstants.PARASOFT_SOATEST_TEST_EXECUTION_REPORT_PATHS_KEY)
+                .name(Messages.SOAtestTestExecutionReportPathName)
+                .description(Messages.SOAtestTestExecutionReportPathDesc)
+                .onQualifiers(Qualifiers.PROJECT)
+                .category(CoreProperties.CATEGORY_CODE_COVERAGE)
```

Comment:
I'll double check with Nathan about whether we should put this under code coverage section.

Reply:
Please put this under "External Analyzers" with subcategory name as "Parasoft SOAtest".